manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml

```diff
@@ -0,0 +1,44 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Waterfox.Waterfox
+PackageVersion: G6.0.3
+PackageLocale: en-US
+Publisher: WaterfoxLimited
+PublisherUrl: https://www.waterfox.net
+PublisherSupportUrl: https://github.com/WaterfoxCo/Waterfox/issues
+PrivacyUrl: https://www.waterfox.net/docs/policies/privacy
+Author: Alexandros Kontos, Adam Wood
+PackageName: Waterfox
+PackageUrl: https://www.waterfox.net
+License: Mozilla Public License
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0
+Copyright: © 2023 Waterfox Ltd, all rights reserved.
```

Comment:
As above, should be BrowserWorks.

Reply:
At least last time I checked this was still what it said on the website

Reply:
Website and browser both list BrowserWorks:
![image](https://github.com/microsoft/winget-pkgs/assets/4728920/66b5bdef-fc3b-40ff-a7cb-1a8294abad05)
![image](https://github.com/microsoft/winget-pkgs/assets/4728920/b51334b3-8b15-4ee9-8719-9a55e98068c4)